core/src/hooks/useAiChatbot.ts

```diff
@@ -0,0 +1,85 @@
+import { useCallback, useContext, useEffect, useRef, useState } from "react";
+
+import { GetAIChatbotsResponse } from "../types/api-responses/getAIChatbotsResponse";
+import LMGlobalClientProviderContext from "../context/LMGlobalClientProviderContext";
+
+export function useAiChatbot() {
+  const { lmChatclient } = useContext(LMGlobalClientProviderContext);
```

Comment:
```suggestion
  const { lmChatClient } = useContext(LMGlobalClientProviderContext);
```
camel case is not follwed

Reply:
done